src/components/Comments.jsx

```diff
@@ -26,6 +29,27 @@ export function Comments({setCommentCount, commentCount}) {
         return <p>No comments yet...</p>
     }
 
+    function handleDelete(commentToRemove) {
+       deleteComments(commentToRemove.comment_id).then(() => {
+        setAlertMessage('Comment Deleted!')
+       })
+       .catch((err)=> {
+        setAlertMessage('Error deleting comment, try again later')
+       })
+       setComments((currComments) => {
```

Comment:
If we are going to optimistically delete, we must undo the deletion in the catch block of the API calling function. If this wasn't meant to be optimistic, you need to do this state change in the then block of the api function.